mod-gobi/schemas/mapping.json

```diff
@@ -588,8 +583,13 @@
               "description": "Lookup acquisition method UUIDs"
             },
             {
-              "name": "LOOKUP_ACQUISITION_UNIT_DEFAULT_ACQ_UNIT_NAME",
-              "title": "lookupAcquisitionUnitDefaultAcqUnitName",
+              "name": "LOOKUP_ACQUISITION_UNIT_DEFAULT",
+              "title": "lookupAcquisitionUnitDefault",
+              "description": "Lookup acquisition unit UUIDs associated with this organizations account"
```

Comment:
Description should be different that in LOOKUP_ACQUISITION_UNIT_BY_ACCOUNT translator

Reply:
fixed.